entity-service-api/build.gradle.kts

```diff
@@ -59,4 +59,8 @@ dependencies {
   api("io.grpc:grpc-protobuf:1.32.1")
   api("io.grpc:grpc-stub:1.32.1")
   api("javax.annotation:javax.annotation-api:1.3.2")
+
```

Comment:
Guava is not a runtime only dep, this constraint should match how the dependency is included. But in general, we can refer snyk failures to a separate PR, no need to handle them with the dockerhub changes

Reply:
`snyk-scan` is a required test and it won't allow merge without fixing it.

Reply:
we can override that in this case, let's not mix changes. The guava change here has not insignificant implications and a decent chance of causing runtime issues (both because we're changing the version at runtime so it differs from compile tie, and because we're using the android version which is missing java 8 support)

Reply:
reverted the snyk related changes. will revert in other repos as well.